zen/models/ChallengeLevel.swift

```diff
@@ -1,5 +1,15 @@
+import Foundation
+
 enum ChallengeLevel: Int, Codable {
     case low = 1
     case medium
     case high
+
+    var description: String {
+        switch self {
+        case .low: return NSLocalizedString("low", comment: "")
```

Comment:
create an extension for NSLocalizedString("low", comment: "")

Reply:
it's a function, not class

Reply:
extenstion for String...

Reply:
it's a function, not class